modules/localization/conf/localization.conf

```diff
@@ -211,3 +211,7 @@
 # need cpu to support avx intel intrinsics
 # default: false
 --if_use_avx=true
+
+--lidar_extrinsics_file=/apollo/modules/localization/msf/params/velodyne_params/velodyne64_novatel_extrinsics_example.yaml
+--map_dir=/apollo/Bag/Test_Data/Sunnyvale_Big_Loop
```

Comment:
Please use the map directory gflag globally across all modules to minimize confusion

Reply:
oh， sorry, it's my mistake.